actors/common/src/main/java/org/sunbird/learner/organisation/external/identity/service/OrgExternalService.java

```diff
@@ -0,0 +1,37 @@
+package org.sunbird.learner.organisation.external.identity.service;
+
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import org.apache.commons.collections.CollectionUtils;
+import org.apache.commons.collections.MapUtils;
+import org.sunbird.cassandra.CassandraOperation;
+import org.sunbird.common.models.response.Response;
+import org.sunbird.common.models.util.JsonKey;
+import org.sunbird.helper.ServiceFactory;
+
+public class OrgExternalService {
+
+  private final String KEYSPACE_NAME = "sunbird";
```

Comment:
KEYSPACE_NAME  and table name need to be configured in one place , that will help us to Change  keyspace name due to any reason.